src/OpenTK.Compute/OpenCL/CL.cs

```diff
@@ -16,6 +16,8 @@ static CL()
 
 		public delegate void ClEventCallback(IntPtr waitEvent, IntPtr userData);
 
+		public static readonly IntPtr ContextPlatform = (IntPtr)0x1084;
```

Comment:
From the documentation this looks like it should actually be an enum.
https://www.khronos.org/registry/OpenCL/sdk/1.0/docs/man/xhtml/clCreateContextFromType.html

Reply:
The issue here is that the enum has to be based on an IntPtr and C# doesn't allow that also when creating a context as properties it expects an array of intptr where the first element should be the property name then the value in the C file I read it seems to have a typedef with intptr as base

Reply:
I feel that is we decide to not make this am enumerated we want to at least note that this is supposed to be an enum in a documentation comment.